pyproject.toml

```diff
@@ -8,7 +8,7 @@ homepage = "https://github.com/Korijn/observ"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.6"
+python = ">=3.6,<3.10"
```

Comment:
At first I didn't like this, but technically it is true, we don't test for python 3.10 and our observable collections definitely need it!